include/ltest-macros.lfe

```diff
@@ -45,14 +45,16 @@
 (defmacro deftest
   "Define a standard EUnit test."
   ((name . body)
-   `(defun ,(list_to_atom (++ (to-unders name) "_test")) ()
-      ,@body)))
+   (let ((name_test (list_to_atom (++ (to-unders name) "_test"))))
```

Comment:
Well, there's no gensym in LFE ... maybe we could name it something stranger, though? More underscores, chars, etc.?